samples/bluetooth/peripheral_uart/Kconfig.sysbuild

```diff
@@ -6,8 +6,19 @@
 
 source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
 
+config NRF_IPC_RADIO_RPC_HOST
+	bool "Enable building RPC Host for Radio domain"
+	help
+	  The option enables building sample rpc_host application for Radio
+	  domain. This sysbuild configuration option should be selected only if
+	  an Application core application is build with CONFIG_BT_RPC_STACK
+	  enabled.
```

Comment:
why do we need yet another kconfig for the same thing?

Reply:
That was my bad, I should have added description to `NRF_DEFAULT_RPC_HOST`.
Anyway the config was removed. Since there is change in approach and `ipc_radio` application will be used instead of `rpc_host` part of the content in this file was extracted to `sysbuild.conf` and `sysbuild_bt_rpc.conf`.